dotCMS/src/main/java/com/dotmarketing/portlets/structure/business/FieldAPIImpl.java

```diff
@@ -24,7 +25,7 @@
  *             interact with Content Type fields.
  */
 @Deprecated
-public class FieldAPIImpl implements FieldAPI {
+public class FieldAPIImpl implements FieldAPI, Serializable {
```

Comment:
Codacy found an issue: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6315945)